app/views/bids/index.html.erb

```diff
@@ -4,7 +4,7 @@
       <div class="user-header-title">
         <h1><%= t 'labels.vendor.account.title' %></h1>
       </div>
-      <%= render partial: 'users/subnav', locals: {active: :bids_placed} %>
+      <%= render partial: 'users/subnav', locals: {subnav: @view_model.subnav_view_model} %>
```

Comment:
it is? it's what we've done in the rest of this project. I am fine with changing but it is pretty common to enforce this in Ruby projs that I've seen.